src/main/java/uk/nhs/digital/ps/migrator/report/DatasetMigrationImpact.java

```diff
@@ -7,7 +7,9 @@
     FIELD_NOT_MIGRATED("Field not migrated"),
     FIELD_MIGRATED_PARTIALLY("Field migrated partially"),
     DATASET_NOT_MIGRATED("Dataset not migrated"),
-    ONE_DUPLICATE_DATASET_MIGRATED("Only one of duplicates migrated");
+    ONE_DUPLICATE_DATASET_MIGRATED("Only one duplicate migrated"),
+    DATASET_MIGRATED_UPDATED_NODE_NAME("Node name updated, dataset migrated")
+    ;
```

Comment:
Does this `;` need to be on new line?

Reply:
No, it doesn't - just missed it when adding new value.

I'll fix it before merging.